holoviews/core/dimension.py

```diff
@@ -677,7 +677,7 @@ def dimensions(self, selection='all', label=False):
         else:
             raise KeyError("Invalid selection %r, valid selections include"
                            "'all', 'value' and 'key' dimensions" % repr(selection))
-        return [(dim.name if label == 'long' else dim.alias)
+        return [(dim.name if label == 'long' else dim.key)
```

Comment:
Now that we have ``label='key'`` I am wondering if the alternative isn't ``label='long'`` but ``label='name'``?